MetaMask/metamask-extension
# feat: update button primary to use light theme on dark mode

## **Description**

This PR updates the `ButtonPrimary` component to use light theme colors when rendered in dark mode. This change is part of the ongoing brand evolution, ensuring that the visual design is consistent with the new brand guidelines. The primary goal is to align the dark mode appearance of this component with the light theme color palette, thereby improving the overall user experience and visual coherence.

## **Related issues**

Fixes: 733

## **Manual testing steps**

1. Navigate to any page where the `ButtonPrimary` component is rendered.
2. Enable dark mode in the application.
3. Observe the color scheme of the `ButtonPrimary` component to ensure it uses the updated light theme colors.
4. Compare the colors with the brand guidelines provided.
5. Test across various instances of the `ButtonPrimary` component in dark mode to ensure consistency.

## **Screenshots/Recordings**

### **Before**

<img width="1323" alt="Screenshot 2024-09-03 at 2 44 28 PM" src="https://github.com/user-attachments/assets/95a535b3-0aec-4d3a-a197-fdd47769838e">

https://github.com/user-attachments/assets/8ea4bf15-abe9-49e6-bde6-17f43d477be4


### **After**
<img width="1152" alt="Screenshot 2024-09-03 at 2 45 14 PM" src="https://github.com/user-attachments/assets/07c54444-1495-4db5-9ea6-d0a2216e4206">



## **Pre-merge author checklist**

- [x] I’ve included tests where applicable.

## **Pre-merge reviewer checklist**
